metadata-io/src/test/java/com/linkedin/metadata/timeline/eventgenerator/SchemaMetadataChangeEventGeneratorTest.java

```diff
@@ -7,16 +7,128 @@
 import com.linkedin.metadata.timeline.data.ChangeEvent;
 import com.linkedin.mxe.SystemMetadata;
 import com.linkedin.restli.internal.server.util.DataMapUtils;
+import com.linkedin.schema.SchemaField;
+import com.linkedin.schema.SchemaFieldArray;
 import com.linkedin.schema.SchemaMetadata;
+import java.net.URISyntaxException;
 import java.nio.charset.StandardCharsets;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import org.apache.commons.io.IOUtils;
 import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
 import org.testng.annotations.Test;
 
 public class SchemaMetadataChangeEventGeneratorTest extends AbstractTestNGSpringContextTests {
 
+  private static Urn getTestUrn() throws URISyntaxException {
+    return Urn.createFromString("urn:li:dataset:(urn:li:dataPlatform:hdfs,SampleHdfsDataset,PROD)");
+  }
+
+  private static AuditStamp getTestAuditStamp() throws URISyntaxException {
+    return new AuditStamp()
+        .setActor(Urn.createFromString("urn:li:corpuser:__datahub_system"))
+        .setTime(1683829509553L);
+  }
+
+  private static void compareDescriptions(
+      Set<String> expectedDescriptions, List<ChangeEvent> actual) {
+    Set<String> actualDescriptions = new HashSet<>();
+    actual.forEach(
+        changeEvent -> {
+          actualDescriptions.add(changeEvent.getDescription());
+        });
+    assertEquals(expectedDescriptions, actualDescriptions);
```

Comment:
There is not `assertEquals` function for sets. This will do object comparison. Do you want to check whether 2 sets are the exact same? Just their contents. as perfect overlapping match? subset/superset?

Reply:
I have tried this and it works as expected.